hardhat.config.ts

```diff
@@ -25,6 +25,7 @@ let config: HardhatUserConfig = {
   networks: {
     hardhat: {
       deploy: ["./deploy/mainnet/"],
+      blockGasLimit: 300000000000000,
```

Comment:
We shouldnt change the default block gas limit since this way we cant easily catch txs that will cost over the prod gas limits.

Reply:
I'm guessing this is due to vyper's ABI containing incorrect gas estimates which will cause some txs to use absurdly high gas limit.

The correct workaround would be to either remove all gas limits in vyper ABI or define realistic gas limit per tx.